backend/api/Areas/Admin/Controllers/UserController.cs

```diff
@@ -68,6 +70,20 @@ public UserController(ILogger<UserController> logger, IConfiguration configurati
             return new JsonResult(paged);
         }
 
+        /// <summary>
+        /// GET - Returns a paged array of users from the datasource that belong to the same agency (or sub-agency) as the current user.
+        /// </summary>
+        /// <param name="page"></param>
+        /// <param name="quantity"></param>
+        /// <param name="sort"></param>
+        /// <returns>Paged object with an array of users.</returns>
+        [HttpGet("/api/admin/my/users")]
+        public IActionResult GetMyUsers(int page = 1, int quantity = 10, string sort = null) // TODO: sort and filter.
+        { 
+            var userId = this.User.GetUserId();
```

Comment:
I think all you're really interested in is the current user's agencies they belong to.  There is an extension method to do this, as we'll include this information in their identity.  Otherwise we have to constantly scan the DB for it.
`User.GetAgency()` returns their primary agency.
`User.GetAgencies()` returns all the agencies they belong to (including child agencies).
I'd probably lean towards creating a new method in the service layer for this rather than just a filter.  It'll allow us to apply security easier.